### PR TITLE
Add information about distribute and virtualenv to the README

### PR DESCRIPTION
**Pull request by niran**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -15,6 +15,11 @@ Getting Started
 
 Installation
 """"""""""""
+If you're installing Armstrong within a `virtualenv`_ (which you should be),
+you'll have better luck if you create your virtualenv with the ``--distribute``
+flag. If you're not using virtualenv, your setuptools installation will be
```
Comment (tswicegood) [inline README.rst]: I'd really like to see this in action before we add a line like this to the docs. It borders on FUD.

Comment (niran) [inline README.rst]: It's not FUD. The distribute docs themselves have a very similar warning (http://pypi.python.org/pypi/distribute#about-the-installation-process). Installing distribute silently without passing on that warning is wrong. Let's fix it.

**Comment by niran** [comment]
This now only includes the `--distribute` recommendation, which is valid.

File: README.rst (hunk 1 above)
Comment (niran) [inline README.rst]: For posterity, we have distribute in arm_sections's setup_requires, but that doesn't install anything system-wide, so nothing meaningful should break. Installing distribute beside setuptools during an install is undefined behavior, hence the remaining recommendation.

**Pull request merged**